Tools/WebKitTestRunner/WebKitTestRunner.xcodeproj/project.pbxproj

```diff
@@ -942,8 +942,8 @@
 				BC25197111D15E61002EBC01 /* InjectedBundle.xcconfig */,
 				57A0062C22976E4D00AD08BD /* WebKitTestRunner.entitlements */,
 				A18510381B9ADF2200744AEB /* WebKitTestRunner.xcconfig */,
+				E3C2C0312C34DEFC000C7F2E /* WebKitTestRunnerApp-iOS-simulator.entitlements */,
 				9B0D132E2036D346008FC8FB /* WebKitTestRunnerApp-iOS.entitlements */,
-				E3660D6A2C2F663800BA7138 /* WebKitTestRunnerApp-visionOS.entitlements */,
```

Comment:
Ditto.  Are you sure that you want to remove `WebKitTestRunnerApp-visionOS.entitlements` from the project?

Reply:
Yes, it has been renamed to WebKitTestRunnerApp-iOS-simulator.entitlements. The intention is to let visionOS use WebKitTestRunnerApp-iOS.entitlements for now.

Thanks for reviewing!